src/game.js

```diff
@@ -0,0 +1,115 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    this.tilesWin = tilesWin;
+    this.length = size * size;
```

Comment:
this.length - it is not quite clear the length of what? I think should use clearer name for this property, maybe `this.boardSize` or smth similar

Reply:
than i use that like the length of my tiles!

Reply:
`this.length` gives not clear vision the length of what yoyu mean here

Reply:
okay, i will change on your version)